Add release drafter, cs fix and auto deployment of the archive via travi

jolelievre opened the pull request:
Add release drafter, cs fix and auto deployment of the archive via travi

jolelievre committed:
Add release drafter, cs fix and auto deployment of the archive via travis

jolelievre committed:
Appliy cs fix

Quetzacoalt91 commented on composer.json:
```diff
@@ -12,6 +12,10 @@
   },
+  "require-dev": {
+    "friendsofphp/php-cs-fixer": "^2.15.1",
+    "prestashop/php-coding-standards": "dev-master"
```
Lib is now 

```suggestion
    "prestashop/php-dev-tools": "^2"
```

Quetzacoalt91 commented on .travis.yml:
```diff
@@ -0,0 +1,48 @@
+cache:
+  directories:
+    - $HOME/.composer/cache
+
```
```suggestion

install:
  - composer install
  
```

jolelievre committed:
Add missing file and correct dependencies

jolelievre committed:
Update lock for compatibility with php 5.6

jolelievre commented:
@Quetzacoalt91 I addressed your modifications, added a few missing files, and the build is now green
Can we merge it in dev without the API key? It is only used for a merge on master right?

Quetzacoalt91 commented:
Yes the API key is needed only when creating tags, this can wait.

Quetzacoalt91 commented:
Thank you @jolelievre
